terraform/domains/environment_domains/config/production.tfvars.json

```diff
@@ -17,6 +17,24 @@
             "to-domain": "claim-funding-for-mentor-training.education.gov.uk"
           }
         ]
+      },
+      "manage-school-placements.education.gov.uk": {
+        "front_door_name": "s189p01-ittmssp-domains-fd",
+        "resource_group_name": "s189p01-ittmsdomains-rg",
+        "domains": [
+          "apex", "www"
+        ],
+        "cached_paths": [
+          "/assets/*"
+        ],
+        "environment_short": "pd",
+        "origin_hostname": "manage-school-placements.teacherservices.cloud",
+        "redirect_rules": [
+          {
+            "from-domain": "www",
+            "to-domain": "claim-funding-for-mentor-training.education.gov.uk"
```

Comment:
"manage-school-placements.education.gov.uk ?

Reply:
Fixed this